src/domain/common/canvas/canvas.entity.ts

```diff
@@ -33,4 +34,12 @@ export class Canvas extends NameableEntity implements ICanvas {
   })
   @JoinColumn()
   checkout?: CanvasCheckout;
+
+  @OneToOne(() => Visual, {
+    eager: true,
+    cascade: true,
+    onDelete: 'SET NULL',
+  })
+  @JoinColumn()
+  bannerCard?: Visual;
```

Comment:
why not just `visual`?

Reply:
Correct, this banner is used on cards - so bannerCard is a far better name

Reply:
This name is coupling the API to this version of the front end

Reply:
bannerNarrow tells nothing, and I have repeatedly seen confusion related to this term. Suggestions for a better name?

Reply:
My first suggestion still stands.